components/ReadMore.js

```diff
@@ -1,15 +1,15 @@
 import react, { Fragment, useState } from "react";
 
 const ReadMore = props => {
-  const [toggleState, setToggleState] = useState("off");
+  const [toggleState, setToggleState] = useState(true);
```

Comment:
I'm thinking we can call toggleState something else? This was placeholder as I was learning react hooks from the docs

Reply:
I've been meaning to write tests for all these things. Will see what I can do today